profiles/fscloud/version.tf

```diff
@@ -0,0 +1,3 @@
+terraform {
+  required_version = ">= 1.3.0"
+}
```

Comment:
You should add ibm provider in here (see https://github.com/terraform-ibm-modules/terraform-ibm-icd-postgresql/blob/main/profiles/fscloud/version.tf)

Reply:
Added